src/libNode/Node.cpp

```diff
@@ -2474,6 +2474,10 @@ bool Node::ToBlockMessage([[gnu::unused]] unsigned char ins_byte) {
                      SyncType::GUARD_DS_SYNC &&
                  GUARD_MODE) {
         return true;
+      } else if (m_mediator.m_lookup->GetSyncType() == SyncType::NORMAL_SYNC &&
```

Comment:
Can we have some log here?

Reply:
we do log the message at caller of `ToBlockMessage` i.e. "Node is not yet in network, so ignore message"